.github/ISSUE_TEMPLATE/cut-release.md

```diff
@@ -20,6 +20,7 @@ see template below, example: https://github.com/kubernetes/sig-release/issues/84
   - [ ] Notify [#release-management](https://kubernetes.slack.com/messages/CJH2GBF7Y): [see notification](https://link-to-slack-message)
   <!-- e.g. https://kubernetes.slack.com/archives/CJH2GBF7Y/p1572365803088800 -->
   - [ ] Collect metrics and add them to the `Release steps` table below
+<!-- ONLY FOR RC RELEASES - [ ] Update the publishing-bot rules to add rules for a new release branch -->
```

Comment:
:thinking: I don't know about listing this one here, there are lots of other things related to the branch creation which have to be performed. We should either have a global __:black_square_button: do branch-creation things__ checkbox or nothing at all, wdyt?

Reply:
@puerco -- whenever you can, can you please enumerate the other things that need to be performed during branch creation that may not be in the handbook?

Reply:
I think in the handbook the list is mostly complete. I think the last time i cut the branch some instructions for test infra were outdated (I noticed @spiffxp fixed some of the generated jobs, for example). But [the list itself](https://github.com/kubernetes/sig-release/blob/master/release-engineering/role-handbooks/branch-manager.md#branch-creation) seems ok.